save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityTableView.kt

```diff
@@ -0,0 +1,371 @@
+/**
+ * View for vulnerability tables
+ */
+
+package com.saveourtool.save.frontend.components.views.vuln
+
+import com.saveourtool.save.entities.cosv.VulnerabilityMetadataDto
+import com.saveourtool.save.filters.VulnerabilityFilter
+import com.saveourtool.save.frontend.TabMenuBar
+import com.saveourtool.save.frontend.components.basic.renderOrganizationWithName
+import com.saveourtool.save.frontend.components.basic.renderUserAvatarWithName
+import com.saveourtool.save.frontend.components.basic.table.filters.vulnerabilitiesFiltersRow
+import com.saveourtool.save.frontend.components.tables.*
+import com.saveourtool.save.frontend.components.views.contests.tab
+import com.saveourtool.save.frontend.externals.i18next.useTranslation
+import com.saveourtool.save.frontend.themes.Colors
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopResponseHandler
+import com.saveourtool.save.info.UserInfo
+import com.saveourtool.save.validation.FrontendRoutes
+
+import js.core.jso
+import react.*
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.td
+import react.dom.html.ReactHTML.th
+import react.dom.html.ReactHTML.tr
+import react.router.dom.Link
+import web.cssom.Border
+import web.cssom.ClassName
+import web.cssom.ColorProperty
+import web.cssom.rem
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+/**
+ * [FC] for [vulnerabilityTableView]
+ */
+val vulnerabilityTableView: FC<VulnerabilityTableViewProps> = FC { props ->
```

Comment:
It's better to rename this to vulnerabilityTableComponent
This is not a view. View has its own unique URL address, which leads to it in our Save.
In this case, this is a component for drawing a table on CollectionView.

Reply:
Done